addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -324,6 +324,39 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
                     continue;
                 }
 
+                // Skip "animation-play-state" changes only
+                if (record.type === 'attributes' && record.attributeName === 'style') {
+                    const newValue = record.target.style.cssText;
+                    const oldAnimationState = (record.oldValue || '').match(/animation-play-state:\s*([a-z\-]+)/);
+                    const newAnimationState = newValue.match(/animation-play-state:\s*([a-z\-]+)/);
+                    if (oldAnimationState && newAnimationState && oldAnimationState[1] !== newAnimationState[1])
+                        continue;
+                }
+
+                // Skip mutations where only animation-related classes are added/removed
+                if (record.type === 'attributes' && record.attributeName === 'class') {
+                    const oldClasses = (record.oldValue || '').split(/\s+/);
+                    const newClasses = [...record.target.classList];
+
+                    const isAnimClass = cls =>
+                        /^o_anim/.test(cls) || [
+                            'o_animated_text',
```

Comment:
Same: is not the point of the editor to track editor changes? If my text becomes animated as per user request, it should be marked dirty